### ISSUE TITLE
[Firefox] Last rows of turbotable with frozen columns are misaligned when there is horizontal scrollbar

### ISSUE THREAD

**Issue by thaihacong** [primefaces/primeng#7654]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
```
Although #6560 was closed, it works only on Chrome, the problem still happens on Firefox.

**Plunkr Case (Bug Reports)**
https://www.primefaces.org/primeng/#/table/scroll > Frozen Columns

**Current behavior**
Scroll down to the bottom of the table with Frozen Columns, last rows are misaligned.

![PrimeNG-table-frozen-columns-scroll-issue](https://user-images.githubusercontent.com/8019304/57368841-71326900-718c-11e9-9f66-9d0f532f8d3c.png)

**Expected behavior**
All rows are aligned no matter the horizontal scrollbar appears or not.

**Please tell us about your environment:**

* **Angular version:** 7.2.8
* **PrimeNG version:** 7.1.2
* **Browser:** Firefox Quantum 60.3.0esr (64-bit) on Centos 7, Firefox 66.0.4 (64-Bit) on Windows 10 Pro

**Comment by longntx** [comment]
If that is not a serious problem to your project, you can manually set the height of 2 tables: frozen and not-frozen heights to archive something like this:
![Screen Shot 2019-06-14 at 10 43 45 AM](https://user-images.githubusercontent.com/13312265/59482091-89835b00-8e91-11e9-8d2d-162bb161286c.png)

**Comment by JobayerAhmmed** [comment]
Add the following code after fetching table data section:

```
setTimeout(() => {
            let frozenBody = document.querySelector(".ui-table-frozen-view > .ui-table-scrollable-body");
            if (frozenBody) {
              let frozenBodyStyle = frozenBody["style"];
              if (frozenBodyStyle["marginBottom"] !== "unset") {
                frozenBodyStyle["paddingBottom"] = frozenBodyStyle["marginBottom"];
                frozenBodyStyle["marginBottom"] = "unset";
              }
            }
          }, 0);
```

**Comment by MassiveHiggsField** [comment]
I have tried your solution @JobayerAhmmed, see here:

https://stackblitz.com/edit/github-gvyvtp

But that just pushes the scroll bar into the pagination.

**Comment by thaihacong** [comment]
Fixed in v9.0.0.
